WordPress/src/main/java/org/wordpress/android/ui/accounts/BlogUtils.java

```diff
@@ -188,4 +188,20 @@ public static void convertToLowercase(Editable s) {
             s.replace(0, s.length(), lowerCase);
         }
     }
+
+    public static Set<String> planTags() {
```

Comment:
Seems that `planTags()` will always return a `HashSet` even if empty. May I suggest adding the `@NonNull` annotation to its return type? Thanks!